authclient/authclient.go

```diff
@@ -27,6 +27,23 @@ func New(options ...Option) *AuthClient {
 	}
 }
 
+func (client *AuthClient) CheckBearerToken(ctx context.Context, serverURL *url.URL, bearerToken string) error {
+	browserURL := getBrowserURL(serverURL)
+	dst := browserURL.ResolveReference(&url.URL{
+		Path: "/api",
```

Comment:
Would `/livez` potentially be a better endpoint to use here?

Reply:
Probably. I will have to test this.